haiku-apps/eventual/eventual-1.0.recipe

```diff
@@ -0,0 +1,68 @@
+SUMMARY="A time management system"
+DESCRIPTION="Eventual is a personal time management system for Haiku. \
+Intended to be a full-featured organizer. "
+HOMEPAGE="https://sourceforge.net/projects/eventual/"
+COPYRIGHT="2011 Alexey Burshtein \
+	2001-2002 Kevin H. Patterson"
+LICENSE="MIT
+	BSD (3-clause)"
+REVISION="1"
+srcGitRev="2fd70537847d3b9712e96ebbc4ff37d5e961b480"
+SOURCE_URI="https://github.com/HaikuArchives/Eventual/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="229cd7fa206da0606d32fc0db1f44a5eee20f0e8428e3dec141e4aab272ead68"
+SOURCE_FILENAME="Eventual-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="Eventual-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 ?x86_64"
+
+PROVIDES="
+	eventual = $portVersion
+	app:Eventual = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:g++
+	cmd:make
+	"
+
+BUILD()
+{
+	sh makelib.sh
+	cd src/Libraries
+	mkdir -p ../lib
+	for LIBDIR in */ ; do
+		LIBDIR=${LIBDIR%?}
+		NAME=lib$LIBDIR.so
+		LIB=$LIBDIR/objects.x86-cc2-release/$NAME
```

Comment:
This looks wrong for x86_64. Maybe don't use makelib.sh and instead call "make OBJ_DIR=objects -C $LIBDIR" for each library.